19/04/22 10:45  GetDataSet Exception Message: A connection was successfully established with the server, but then an error occurred during the login process. (provider: TCP Provider, error: 0 - An existing connection was forcibly closed by the remote host.)
. InnerException: An existing connection was forcibly closed by the remote host
. extraInfo: sql: SELECT problemsClose.id, problemsClose.workerId, problemsClose.phoneId, problemsClose.phone, problemsClose.ip, problemsClose.placeNameId, problemsClose.placeName, problemsClose.customerName, problemsClose.problemDesc, problemsClose.problemSolution, problemsClose.statusId, problemsClose.emergencyId, problemsClose.departmentId, problemsClose.reportToYaron, problemsClose.startTime, problemsClose.finishTime, problemStatus.statusName, workers.firstName + N' ' + workers.lastName AS workerName, departments.departmentName, emergencyTypes.emergencyName, problemsClose.toWorker, workers_1.firstName + N' ' + workers_1.lastName AS toWorkerName, t.problemId as pFileId FROM problemsClose LEFT JOIN workers ON problemsClose.workerId = workers.id INNER JOIN problemStatus ON problemsClose.statusId = problemStatus.id INNER JOIN  emergencyTypes ON problemsClose.emergencyId = emergencyTypes.id INNER JOIN  departments ON problemsClose.departmentId = departments.id LEFT JOIN  workers AS workers_1 ON problemsClose.toWorker = workers_1.id LEFT JOIN (SELECT [problemId] FROM [dbo].[problemFiles] Group by problemId) as t on problemsClose.id = t.problemId WHERE statusId=0 AND departmentId=3 ORDER BY problemsClose.startTime DESC

19/04/22 13:17  GetDataSet Exception Message: Invalid column name 'dawd'.
. extraInfo: sql: SELECT problemsClose.id, problemsClose.workerId, problemsClose.phoneId, problemsClose.phone, problemsClose.ip, problemsClose.placeNameId, problemsClose.placeName, problemsClose.customerName, problemsClose.problemDesc, problemsClose.problemSolution, problemsClose.statusId, problemsClose.emergencyId, problemsClose.departmentId, problemsClose.reportToYaron, problemsClose.startTime, problemsClose.finishTime, problemStatus.statusName, workers.firstName + N' ' + workers.lastName AS workerName, departments.departmentName, emergencyTypes.emergencyName, problemsClose.toWorker, workers_1.firstName + N' ' + workers_1.lastName AS toWorkerName, t.problemId as pFileId FROM problemsClose LEFT JOIN workers ON problemsClose.workerId = workers.id INNER JOIN problemStatus ON problemsClose.statusId = problemStatus.id INNER JOIN  emergencyTypes ON problemsClose.emergencyId = emergencyTypes.id INNER JOIN  departments ON problemsClose.departmentId = departments.id LEFT JOIN  workers AS workers_1 ON problemsClose.toWorker = workers_1.id LEFT JOIN (SELECT [problemId] FROM [dbo].[problemFiles] Group by problemId) as t on problemsClose.id = t.problemId WHERE statusId<>2 AND departmentId=dawd ORDER BY problemsClose.startTime DESC

19/04/22 13:46  GetDataSet Exception Message: The client was unable to establish a connection because of an error during connection initialization process before login. Possible causes include the following:  the client tried to connect to an unsupported version of SQL Server; the server was too busy to accept new connections; or there was a resource limitation (insufficient memory or maximum allowed connections) on the server. (provider: TCP Provider, error: 0 - An existing connection was forcibly closed by the remote host.)
. InnerException: An existing connection was forcibly closed by the remote host
. extraInfo: sql: SELECT statusId, SUM(1) AS pCount FROM problemsClose WHERE (toWorker = @workerId) AND (statusId=0 OR statusId=1) GROUP BY statusId;

